src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -64,58 +64,43 @@
 
 
 (defn render-block
-  [blocks {:keys [type ^js literal children]} edited-at]
+  [blocks {:keys [type ^js literal children]}]
   (case (keyword type)
     :paragraph
     (conj blocks
-          (reduce
-           render-inline
-           [quo/text]
-           children))
+          [rn/view {:style style/paragraph}
+           (reduce
+            render-inline
+            [quo/text]
+            children)])
 
     :blockquote
     (conj blocks
           [rn/view {:style style/quote}
-           [quo/text literal]]
-          (when edited-at
-            [quo/text
-             {:weight :medium
-              :style  {:font-size  11
-                       :margin-top 4
-                       :color      (colors/theme-colors colors/neutral-40 colors/neutral-50)}}
-             (str " (" (i18n/label :t/edited) ")")]))
+           [quo/text literal]])
 
     :codeblock
     (conj blocks
           [rn/view {:style (merge style/block (style/code))}
-           [quo/text (subs literal 0 (dec (count literal)))]]
-          (when edited-at
-            [quo/text
-             {:weight :medium
-              :style  {:font-size  11
-                       :margin-top 4
-                       :color      (colors/theme-colors colors/neutral-40 colors/neutral-50)}}
-             (str " (" (i18n/label :t/edited) ")")]))
+           [quo/text (subs literal 0 (dec (count literal)))]])
     blocks))
 
-(defn add-edited-tag
-  [parsed-text]
-  (update parsed-text
-          (dec (count parsed-text))
-          (fn [last-literal]
-            (update last-literal
-                    :children
-                    conj
-                    {:literal (str " (" (i18n/label :t/edited) ")")
-                     :type    :edited}))))
```

Comment:
Can you elaborate on the removal of this function ?

Reply:
Well, edited tag can be added directly when there is a field `edited-at` in the provided text — no need to call `update` inside `update` to add the tag explicitly.

Reply:
Design requirements are to add the tag right after the last piece of text .. Inline if possible

Reply:
Ok well, if the requirement was to write it inline if possible then check the latest commit, now it should be good.

Reply:
Great work Alex!
Thank you
I'll review it in a bit